db/dcrsqlite/apisource.go

```diff
@@ -1309,42 +1309,60 @@ func (db *wiredDB) GetExplorerTx(txid string) *explorer.TxInfo {
 
 func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) (*explorer.AddressInfo, error) {
 	addr, err := dcrutil.DecodeAddress(address)
-	if err != nil {
-		return nil, err
+
+	// checks for P2Pk type addresses
+	if strings.HasPrefix(address, "Dk") || strings.HasPrefix(address, "Tk") || strings.HasPrefix(address, "Sk") {
+		return nil, fmt.Errorf("P2PK address detected")
 	}
+	if err != nil {
+		log.Infof("Invalid address %s: %v", address, err)
 
-	// Short circuit the transaction and balance queries if the provided address
-	// is the zero pubkey hash address commonly used for zero value
-	// sstxchange-tagged outputs.
-	isDummyAddress := IsZeroHashP2PHKAddress(address, db.params)
-	if isDummyAddress {
-		return &explorer.AddressInfo{
-			Address:         address,
-			Balance:         new(explorer.AddressBalance),
-			UnconfirmedTxns: new(explorer.AddressTransactions),
-			IsDummyAddress:  true,
-			Fullmode:        true,
-		}, nil
+		// This is here to detect a bitcoin type address
+		if (strings.HasPrefix(address, "bc") || strings.HasPrefix(address, "1") || strings.HasPrefix(address, "3")) && len(address) >= 25 && len(address) <= 34 {
+			return nil, fmt.Errorf("Possible Bitcoin address detected.")
+		}
+		return nil, nil
```

Comment:
This needs to return a non-nil `error` or the address page will try to display for whatever this is.